tests/baselines/reference/contextualTyping11.errors.txt

```diff
@@ -1,11 +1,10 @@
-tests/cases/compiler/contextualTyping11.ts(1,20): error TS2322: Type 'foo[]' is not assignable to type '{ id: number; }[]'.
-  Type 'foo' is not assignable to type '{ id: number; }'.
-    Property 'id' is missing in type 'foo'.
+tests/cases/compiler/contextualTyping11.ts(1,42): error TS2322: Type 'foo' is not assignable to type '{ id: number; }'.
+  Property 'id' is missing in type 'foo'.
 
 
 ==== tests/cases/compiler/contextualTyping11.ts (1 errors) ====
     class foo { public bar:{id:number;}[] = [<foo>({})]; }
-                       ~~~
-!!! error TS2322: Type 'foo[]' is not assignable to type '{ id: number; }[]'.
-!!! error TS2322:   Type 'foo' is not assignable to type '{ id: number; }'.
-!!! error TS2322:     Property 'id' is missing in type 'foo'.
\ No newline at end of file
+                                             ~~~~~~~~~
+!!! error TS2322: Type 'foo' is not assignable to type '{ id: number; }'.
+!!! error TS2322:   Property 'id' is missing in type 'foo'.
+!!! related TS6501 /.ts/lib.es5.d.ts:1284:5: The expected type comes from this index signature.
```

Comment:
Unfortunately I think that these aren't very helpful. Is there a way to retain the topmost type and point to that?